### ISSUE TITLE
Fault 1: '<class \'Exception\'>:method "info" is not supported'

### ISSUE THREAD

**Issue by PhunkyBob** [unoconv/unoserver#130]
When I try to make a convertion, I have the following message: 

> xmlrpc.client.Fault: <Fault 1: '<class \'Exception\'>:method "info" is not supported'>

## Environment
- Windows 11
- Python 3.11.9
- Pip 24.2
- Unoserver 2.2.1
- Docker with unoserver 2.2.1

## Steps to reproduce

- Run a Docker environment with https://github.com/unoconv/unoserver-docker version 2.2.1.

```
docker run -p 2003:2003 ghcr.io/unoconv/unoserver-docker
```

- Install unoserver with pip in a virtual env.
```
(venv) λ unoconvert --version
unoconvert 2.2.1
```

- Create a word document `test.docx`.
- Run the following command:
```
unoconvert --host-location remote test.docx out.pdf
```

## Expected result
A file `out.pdf` is created. 

## Actual result
```bash
(venv) λ unoconvert --host-location remote test.docx out.pdf
INFO:unoserver:Connecting.
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\PhunkyBob\working_dir\venv\Scripts\unoconvert.exe\__main__.py", line 7, in <module>
  File "C:\Users\PhunkyBob\working_dir\venv\Lib\site-packages\unoserver\client.py", line 354, in converter_main
    result = client.convert(
             ^^^^^^^^^^^^^^^
  File "C:\Users\PhunkyBob\working_dir\venv\Lib\site-packages\unoserver\client.py", line 124, in convert
    info = self._connect(proxy)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\PhunkyBob\working_dir\venv\Lib\site-packages\unoserver\client.py", line 52, in _connect
    info = proxy.info()
           ^^^^^^^^^^^^
  File "C:\Users\nicolas.martin\AppData\Local\Programs\Python\Python311\Lib\xmlrpc\client.py", line 1122, in __call__
    return self.__send(self.__name, args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\nicolas.martin\AppData\Local\Programs\Python\Python311\Lib\xmlrpc\client.py", line 1464, in __request
    response = self.__transport.request(
               ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\nicolas.martin\AppData\Local\Programs\Python\Python311\Lib\xmlrpc\client.py", line 1166, in request
    return self.single_request(host, handler, request_body, verbose)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\nicolas.martin\AppData\Local\Programs\Python\Python311\Lib\xmlrpc\client.py", line 1182, in single_request
    return self.parse_response(resp)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\nicolas.martin\AppData\Local\Programs\Python\Python311\Lib\xmlrpc\client.py", line 1354, in parse_response
    return u.close()
           ^^^^^^^^^
  File "C:\Users\nicolas.martin\AppData\Local\Programs\Python\Python311\Lib\xmlrpc\client.py", line 668, in close
    raise Fault(**self._stack[0])
xmlrpc.client.Fault: <Fault 1: '<class \'Exception\'>:method "info" is not supported'>
```

- On Docker side, the request is displayed: 

> 172.17.0.1 - - [04/Sep/2024 16:56:01] "POST /RPC2 HTTP/1.1" 200 -

**Comment by regebro** [comment]
You are running an older version of the unoserver package for the server
and a new one for the client. Make sure you run the same version for both.

**Comment by PhunkyBob** [comment]
I'm not sure to understand.

In `Dockerfile`:
```
ARG VERSION_UNOSERVER=2.2.1
```

And in local: 
```
(venv) λ unoconvert --version
unoconvert 2.2.1
```

Unoserver 2.2.1 is not compatible with unoconvert 2.2.1 ?

**Comment by PhunkyBob** [comment]
OK, you were right. 
The image I used was not OK.

**Comment by jorge-jrzz** [comment]
> OK, you were right. The image I used was not OK.
Wait, I have the same problem, how did you solve it?

**Comment by PhunkyBob** [comment]
> Wait, I have the same problem, how did you solve it?

I thought I was using the image I build, but I was using a remote image (not up to date).
The is is I'm using Windows and when I built the image from repo, I had an error. Something related to the endpoint. 
To correct this error, I had to open the endpoint.sh file with Notepad++ and replace "Windows LF" by "Linux LF" so that the system don't see "^M" characters at the end of all lines.
After that, I could built the image and use it.
